company-profiles/featurist.md

```diff
@@ -2,10 +2,28 @@
 
 ## Company blurb
 
-⚠ We don't have much information about this company yet!
+Featurist provides full stack, feature driven development teams. They build mobile and web apps, transform legacy systems and contribute to the open-source community.
 
-If you know something we don't, help us fill it in!  Here's how:
+## Company size
 
-- Read our [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md)
-- Have a look at our [example company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md)
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/featurist.md)
+0-20.
+
+## Remote status
+
+Remote workers are welcomed.
+
+## Region
+
+Worldwide.
+
+## Company technologies
+
+NA
+
+## Office locations
+
+NA
```

Comment:
Let's leave these two sections out instead of writing NA.

Reply:
Ah the bot has been very helpful here! I understand now that I should delete the sections with no information. Thanks!